extensions/reviewed/Flash.json

```
@@ -909,6 +913,87 @@
             }
           ],
           "objectGroups": []
+        },
+        {
+          "description": "the half period (time during which object is invisible) of the object.",
+          "fullName": "Half period (time during which object is invisible)",
```

```suggestion
          "fullName": "Half period",
```
Names should not include description the description field is enough.

---

The issue is probably that the property name is also too long and the property description should have been used.
The generator automatically adds the property description at the end of the expression description.